swarm-production/docker-stack.yml

```diff
@@ -1,12 +1,12 @@
 ---
 
-version: "3.1"
+version: "3.4"
 
 services:
   s3-data:
-    image: scality/s3server
+    image: zenko/cloudserver:pensieve-0
```

Comment:
are we going to maintain 2 docker images for open source?

Reply:
does your image have S3DATA=multiple by default?  how will people configure the locations without pensieve?

Reply:
S3DATA=multiple is hardcoded because it would not work with Orbit otherwise. The gradual switch from the old image to the new one was decided for Orbit - the source dockerfile is still the same.

Reply:
as discussed, all of the locations are file at start up.